charts/hono/templates/hono-adapter-amqp/headless-svc.yaml

```diff
@@ -1,16 +0,0 @@
-{{- if and .Values.adapters.amqp.enabled .Values.prometheus.createInstance }}
```

Comment:
The headless services should then also be removed from the `hono/README.md` file.

Btw., what is the remaining `eclipse-hono-service-device-registry-headless` service used for?

Reply:
exactly